admin_tests/preprint_providers/test_views.py

```diff
@@ -219,6 +229,59 @@ def test_certain_fields_not_included(self):
         for field in views.FIELDS_TO_NOT_IMPORT_EXPORT:
             nt.assert_not_in(field, content_dict['fields'].keys())
 
+    def test_export_to_import_new_provider(self):
+        update_taxonomies('test_bepress_taxonomy.json')
```

Comment:
Similarly to the management command concern, this is a little weird. We should be able to send a dictionary to a function to update the taxonomy, and then we could call a test without having to make a test file. Anyways, no change here, just thinking.